content/tutorials/grafana-fundamentals.md

```diff
@@ -75,27 +88,27 @@ The sample application, Grafana News, lets you post links and vote for the ones
 
 To add a link:
 
-- In the **Title** box, type "Example".
-- In the **URL** box, type "https://example.com".
+- In **Title**, enter **Example**.
```

Comment:
```suggestion
- In **Title**, enter "Example".
```

As I understand the style guide, when referring to strings/text for input, we should use quotation marks.

Under **Strings** in the [style guide](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions)

Reply:
Under **User Input**, it recommends bold or italic.